src/overlays/actors/ovl_Bg_Icefloe/z_bg_icefloe.c

```diff
@@ -5,6 +5,7 @@
  */
 
 #include "z_bg_icefloe.h"
+#include "objects/object_icefloe/object_icefloe.h"
```

Comment:
There's only three things to name in this object, you might as well just do it now.